### PR TITLE
Update zh_cn.json and zh_tw.json

### PR DESCRIPTION
**Pull request by buiawpkgew1**

### CODE REVIEW

File: src/main/resources/assets/clientcommands/lang/zh_cn.json [hunk 1]
```diff
@@ -77,6 +77,7 @@
   "commands.cfish.error.impossibleLoot": "那个战利品不能通过钓鱼获得",
   "commands.cfish.error.onGround": "浮标撞到了地上",
   "commands.cfish.error.openWater": "为了获得宝藏战利品，浮标需要在开放的水中",
+  "commands.cfish.error.openWater.flowingWater": "帮助：锅炉周围的水必须仍然是水，水流不起作用。",
```
Comment (Earthcomputer) [inline src/main/resources/assets/clientcommands/lang/zh_cn.json]: > 帮助

The rest of them say "建议", are you sure this is okay?

File: src/main/resources/assets/clientcommands/lang/zh_cn.json [hunk 2]
```diff
@@ -138,15 +139,17 @@
...
   "commands.crelog.failed": "未能重新登录",
 
   "commands.crender.entities.success": "实体渲染规则已更新",
 
   "commands.cstopsound.success.sourceless.any": "已为自身停止播放所有声音",
   "commands.cstopsound.success.sourceless.sound": "已为自身停止播放%s声音",
   "commands.cstopsound.success.source.any": "已为自身停止播放所有%s声音",
-  "commands.cstopsound.success.source.sound": "已为自身停止播放来自%2$s音源的%$1s声音",
 
+  "commands.cstopsound.success.source.sound": "已为自身停止播放来自%s音源的%s声音",
```
Comment (Earthcomputer) [inline src/main/resources/assets/clientcommands/lang/zh_cn.json]: Why did you change this?

File: src/main/resources/assets/clientcommands/lang/zh_cn.json (hunk 1 above)
Comment (buiawpkgew1) [reply src/main/resources/assets/clientcommands/lang/zh_cn.json:80]: okay

**Comment by Earthcomputer** [comment]
You didn't address my second review comment

**Comment by buiawpkgew1** [comment]
> You didn't address my second review comment

**Comment by Earthcomputer** [comment]
Hi @buiawpkgew1

It appears that you have made a project on crowdin using the name clientcommands, but now https://crowdin.com/project/clientcommands redirects to your translations of a different project, YimMenu. I would now like to use the name "clientcommands" on crowdin for clientcommands translations. Would you be so kind as to rename this crowdin project so that I can make an official clientcommands translations project, or if that is not possible, transfer ownership of the repurposed clientcommands crowdin project to me (email burtonjae@hotmail.co.uk), and create a new project for YimMenu?

Thanks in advance

**Comment by Earthcomputer** [comment]
Looks like you might have to delete and recreate the project unfortunately. If this is okay with you, let me know when you have done this and I will create the new clientcommands project.
